tests/software/test_main.py

```diff
@@ -31,3 +33,33 @@ def test_help(self):
         # Basic test. This should run without loading the driver.
         m = chipsec_main.main(['--help'])
         self.assertEqual(ExitCode.OK, m)
+
+    def test_module_enumeration(self):
+        """Run chipsec_main --enum_tools"""
+        module_helper.enumerate_modules()
+
+    def test_read_files_in_tool_folder(self):
+        # default_options = {'help': False, '_module': None, '_module_exclude': None, '_module_argv': None, 'verbose': False, 'hal': False, 'debug': False, 'log': None, 'vverbose': False, '_platform': None, '_pch': None, '_no_driver': False, '_ignore_platform': False, '_json_out': None, '_xml_out': None, '_markdown_out': None, 'USER_MODULE_TAGS': [], '_list_tags': False, 'IMPORT_PATHS': [], 'failfast': False, 'no_time': False, '_deltas_file': None, '_helper': None, '_show_banner': True, '_load_config': True, '_autolog_disable': False, '_return_codes': False}
+        # chipsecMain = chipsec_main.ChipsecMain(default_options, [])
+        res = module_helper.enumerate_modules()
+        expected = ['tools.generate_test_id', 'tools.wsmt', 'tools.cpu.sinkhole', 'tools.secureboot.te', 'tools.smm.rogue_mmio_bar', 'tools.smm.smm_ptr', 'tools.uefi.reputation', 'tools.uefi.s3script_modify', 'tools.uefi.scan_blocked', 'tools.uefi.scan_image', 'tools.uefi.uefivar_fuzz', 'tools.vmm.common', 'tools.vmm.cpuid_fuzz', 'tools.vmm.ept_finder', 'tools.vmm.hypercallfuzz', 'tools.vmm.iofuzz', 'tools.vmm.msr_fuzz', 'tools.vmm.pcie_fuzz', 'tools.vmm.pcie_overlap_fuzz', 'tools.vmm.venom', 'tools.vmm.hv.define', 'tools.vmm.hv.hypercall', 'tools.vmm.hv.hypercallfuzz', 'tools.vmm.hv.synth_dev', 'tools.vmm.hv.synth_kbd', 'tools.vmm.hv.vmbus', 'tools.vmm.hv.vmbusfuzz', 'tools.vmm.vbox.vbox_crash_apicbase', 'tools.vmm.xen.define', 'tools.vmm.xen.hypercall', 'tools.vmm.xen.hypercallfuzz', 'tools.vmm.xen.xsa188']
+        self.assertEqual(res, expected)
```

Comment:
This test is failing as I changed the enumerate_modules from just returning the tools to everything in the modules folder. Should probably change this assertEqual to:
```suggestion
        for exp in expected:
            self.assertTrue(exp in res)
```